packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/helpers/isPrecededByExceptionSpec.js

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line max-len
+import { isPrecededByException, isNotPrecededByException } from "../../../../../src/scoring/assessments/inclusiveLanguage/helpers/isPrecededByException";
+
+xdescribe( "Test isFollowedByException", () => {
```

Comment:
I added specs for isPrecededByException. These specs are currently skipped (xdescribe), because their behaviour is flipped. (isPrecededByException returns false if a word is preceded by an exception.). The specs reflect the desired behaviour of the function. They will be fixed in [this issue](https://yoast.atlassian.net/browse/PC-1008?atlOrigin=eyJpIjoiYTllZjJjNzA5MmMxNDAzOWEzMmUzNDJhNWI3YTNiMDMiLCJwIjoiaiJ9)